http/src/main/java/io/micronaut/http/reactive/execution/ReactorExecutionFlowImpl.java

```diff
@@ -164,6 +172,45 @@ public void onComplete() {
         });
     }
 
+    @Override
+    public void completeTo(CompletableFuture<Object> completableFuture) {
+        if (value instanceof Fuseable.ScalarCallable callable) {
+            try {
+                completableFuture.complete(callable.call());
+            } catch (Exception e) {
+                completableFuture.completeExceptionally(e);
+            }
+            return;
+        }
+        value.subscribe(new CoreSubscriber<>() {
+
+            Subscription subscription;
+            Object value;
+
+            @Override
+            public void onSubscribe(Subscription s) {
+                this.subscription = s;
+                s.request(1);
```

Comment:
use Long.MAX_VALUE, and dont request again in onNext

Reply:
Originally copied from the other method, corrected both